Userland/Applications/FileManager/PropertiesWindow.cpp

```diff
@@ -159,6 +161,9 @@ PropertiesWindow::PropertiesWindow(const String& path, bool disable_rename, Wind
     m_apply_button->on_click = [this](auto) { apply_changes(); };
     m_apply_button->set_enabled(false);
 
+    if (is_hashable(m_mode)) {
+        tab_widget.add_tab<HashTab>("Hashes", path, ((u64)st.st_size) < 2 * MiB);
+    }
```

Comment:
Don't use brackets for single line if.

```suggestion
    if (is_hashable(m_mode))
        tab_widget.add_tab<HashTab>("Hashes", path, ((u64)st.st_size) < 2 * MiB);

```